src/core/stream_recv.c

```diff
@@ -162,6 +162,39 @@ QuicStreamRecvQueueFlush(
     }
 }
 
+//
+// Processes a received RELIABLE_RESET frame's payload.
+//
+_IRQL_requires_max_(PASSIVE_LEVEL)
+void
+QuicStreamProcessReliableResetFrame(
+    _In_ QUIC_STREAM* Stream,
+    _In_ uint64_t FinalSize,
+    _In_ QUIC_VAR_INT ErrorCode,
+    _In_ QUIC_VAR_INT ReliableOffset
+    )
+{
+    if (Stream->ReliableOffsetRecv == 0 || ReliableOffset < Stream->ReliableOffsetRecv) {
+        //
+        // As outlined in the spec, if we receive multiple CLOSE_STREAM frames, we only accept strictly
+        // decreasing offsets.
+        //
+        Stream->ReliableOffsetRecv = ReliableOffset; // TODO: Remove ReliableOffsetRecv once everything works.
+        Stream->RecvMaxLength = ReliableOffset;
```

Comment:
We should add a log too. Something like:
```c
QuicTraceLogStreamInfo(
    ReliableRecvOffsetSet,
    Stream,
    "Reliable recv offset set to %llu",
    ReliableOffset);
```

Reply:
Added